osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -865,114 +909,16 @@ - (void)showPreferences {
     [window setLevel:NSFloatingWindowLevel];
 }
 
-
-- (NSString *)getUsernamePref {
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    NSString *username = [prefs stringForKey:@"username"];
-    return username;
-}
-
-
 - (void)loadPreferences {
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    self.username = [self getUsernamePref];
-
-    // TODO(cpauya): We must encrypt the password!
-    NSString *password = [prefs stringForKey:@"password"];
-    if (password != nil || [password isNotEqualTo:@""]) {
-        // NSData from the Base64 encoded str
-        NSData *nsdataFromBase64String = [[NSData alloc]
-                                          initWithBase64EncodedString:password options:0];
-        // Decoded NSString from the NSData
-        NSString *decodePassword = [[NSString alloc]
-                                    initWithData:nsdataFromBase64String encoding:NSUTF8StringEncoding];
-        self.password = decodePassword;
-        self.confirmPassword = decodePassword;
-    }
+    // TODO(richard): Set custom database path in the preferences menu.
```

Comment:
@amodia Please use your github username in your TODO comments:  Example: `// TODO(amodia)...`.